sockeye/encoder.py

```diff
@@ -455,7 +455,7 @@ def encode_positions(self,
         sin = mx.sym.sin(scaled_positions)
         cos = mx.sym.cos(scaled_positions)
 
-        # (batch_size, num_embed/2)
+        # (batch_size, num_embed)
```

Comment:
Was that a mistake earlier? It shouldn't change with this PR, should it?

Reply:
yeah the comment was wrong